### PR TITLE
Improve READMEs, fix typos

### PR DESCRIPTION
**Pull request by martin-t**
Let me know if you disagree with anything.

The logo might look better if it was a bit smaller but i haven't looked at how to edit the svg.

I changed the heading to "Rust Game 3D" because as i said on discord, it's not immediately obvious what RG3D stands for and i think this way it's more memorable.

There's also a way to center the heading/logo if you think it would look better.

What do you think about including the number of comments https://tokei.rs/b1/github/mrDIMAS/rg3d?category=comments among the badges?

I changed the bit about ECS/OOP because i think a lot of people imagine inheritance when they hear OOP and it might not evoke positive emotions, especially in the rust crowd ;)

### CODE REVIEW

**Comment by toyboot4e** [comment]
Also, I wonder if we substitute WARNING to NOTE in sub crates' README.md so that they have positive nuance :)

**Comment by martin-t** [comment]
Definitely, i didn't notice the subcrates has separate readmes. I skimmed them and made some more fixes.

File: README.md [hunk 1]
```diff
@@ -1,13 +1,14 @@
+[![RG3D](pics/logo2.svg)](https://rg3d.rs/)
+
+# Rust Game 3D
```
Comment (mrDIMAS) [inline README.md:3]: It should be **R**ust **G**ame engine **3D**, I think it is less confusing than Rust Game - which looks like it is game, not engine.

**Review by mrDIMAS — CHANGES_REQUESTED** [review]
Could you please change that one comment? The rest looks good, it is a shame that I missed so many articles :smile:

File: README.md (hunk 1 above)
Comment (mrDIMAS) [reply README.md:3]: Actually, I just realized that I can fix it by myself after merge

**Pull request merged**